api/src/wfl/util.clj

```diff
@@ -108,6 +108,9 @@
     (apply merge-with deep-merge a maps)
     (apply merge-with deep-merge maps)))
 
+(defn when->   [x p f] (if     (p x) (f x) x))
+(defn unless-> [x p f] (if-not (p x) (f x) x))
```

Comment:
These are not really flow macros like `->` or `cond->`.
OK, but pretty sure I do not like them.  (TIHI?)

Reply:
I completely agree, though not sure what TIHI means...
maybe `apply-when` or `return-unless`? IDK - suggestions welcome.